tools/pot/tests/test_nightly.py

```diff
@@ -52,7 +52,7 @@
 TEST_MODELS_ACC_AWARE = [
     ('efficientnet-b0-pytorch', 'pytorch', 'CPU', {'performance': {'accuracy@top1': 0.7663,
                                                                    'accuracy@top5': 0.9294}}),
-    ('mobilenet-ssd', 'caffe', 'CPU', {'performance': {'map': 0.7222}}),
+    ('mobilenet-ssd', 'caffe', 'CPU', {'performance': {'map': 0.67}}),
```

Comment:
Why did it change so much?

Reply:
This is not the same model that was before. This accuracy is normal for the model from OMZ.